configuration/pih/pih-config-mirebalais-humci.json

```diff
@@ -75,6 +75,21 @@
     "matchUrl": "http://localhost:9000/match",
     "scanUrl": "https://localhost:9443/fingerprint/scan",
     "devicesUrl": "https://localhost:9443/fingerprint/devices"
-  }
+  },
+
+  "extensions": [
+    {
+      "id": "patientHeader.linkToHivEmr",
+      "type": "link",
+      "url": "http://localhost:8080/openmrs/pihcore/patient.page?patientId={{identifier}}&identifierType=${patientIdentifierType.hivemrV1.uuid}&dashboard=${program.hiv.uuid}",
+      "extensionPointId": "patientHeader.extraIdentifierLinks",
+      "label": "pihcore.hiv.viewPatientInHivEmr",
+      "icon": "icon-external-link",
+      "requiredPrivilege": null,
+      "extensionParams": {
+        "identifierType": "${patientIdentifierType.hivemrV1.uuid}"
+      }
+    }
+  ]
```

Comment:
I assume it was intentional to only set this up on HUM-CI for now?

Reply:
Actually, probably should have this link to haitihivtest (instead of localhost) so we can test how this actually works?

Reply:
I'll hard-code that in for now, but generally feel like this is something we need to find a way to make come from an external source that can be set via puppet, maybe a runtime property.  I don't love having this hard coded here.